nvaccess/nvda
# Ensure the mouse does not play audio coordinates in sleep mode

### Link to issue number:

Fixes #8059

### Summary of the issue:

This issue seems to be because in the core pump, mouse movement events are being handled, but when executing the mouse movement event, there is no check for whether it is in sleep mode. And the function to play audio coordinates is executed accordingly.

### Description of user facing changes

The audio coordinates are no longer played when the mouse moves over an application in sleep mode.

### Description of development approach

Simultaneously determine whether to report audio coordinates and whether to be in sleep mode.

### Testing strategy:

1. Open mouse settings, and check Play audio coordinates when mouse moves, then hit okay.
2. Open sleep mode on an application

Do not play audio coordinates when moving the mouse on this application

### Known issues with pull request:

I was considering the possibility of failure in the logic of updating the mouse object.

So to be on the safe side I chose to use the old mouse object for the judgment.

Although this will delay the response of a mouse object, I think it is acceptable for this PR.

### Code Review Checklist:



  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
  - Unit tests
  - System (end to end) tests
  - Manual testing
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
